config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  #User Controller
+  resources :users, only: [:index, :show]
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  #Posts Controller
+  resources :posts, only: [:index, :show]
```

Comment:
- [ ] Kindly update your routes to be:
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
instead of 
```ruby
  resources :users, only: [:index, :show]

  resources :posts, only: [:index, :show]
```
as requested by the previous reviewer. Your route now will not make posts route child routes of users routes as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#make-sure-that-when-you-run-the-rails-routes-command-in-your-console-you-can-see-the-following-output:~:text=Make%20sure%20that%20when%20you%20run%20the%20rails%20routes%20command%20in%20your%20console%2C%20you%20can%20see%20the%20following%20output%3A).